io.otsaloma.nfoview.yml

```diff
@@ -0,0 +1,23 @@
+app-id: io.otsaloma.nfoview
+runtime: org.gnome.Platform
+runtime-version: "3.30"
+sdk: org.gnome.Sdk
+command: nfoview
+rename-appdata-file: nfoview.appdata.xml
+rename-desktop-file: nfoview.desktop
+rename-icon: nfoview
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --share=network
```

Comment:
What does it network access for?
Also given it uses GTK3, it should also work on Wayland.

Reply:
Indeed, network isn't used, removed now and Wayland added.